### ISSUE TITLE
Laguages dropdown

### ISSUE THREAD

**Issue by tesseralis** [reactjs/reactjs.org#1772]
While our current [Languages page](https://reactjs.org/languages) is great, a dropdown would be better! This would allow us to switch between different translations of a page quickly. For example, if you get a link to an english version of a page, you can click on the dropdown to go to a different translation of that page instead of going to a separate page and losing your url.

**Comment by nutboltu** [comment]
@tesseralis is there any wireframe for this dropdown?

**Comment by tesseralis** [comment]
Nope, not yet. I think @bvaughn and @ericnakagawa are too busy for it right now.

**Comment by tesseralis** [comment]
I also don't know who's actually responsible for designing the site.🤷‍♀️

**Comment by Italox** [comment]
Can I try it? I think when change language and stay on the same page is better than redirect for initial URL.

**Comment by tesseralis** [comment]
@Italox I already have a version of this here: https://github.com/reactjs/reactjs.org/pull/2255

**Comment by Italox** [comment]
@tesseralis , Oh! Sorry. Very nice implementation, waiting merge in production.
